azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -288,7 +292,7 @@ public void testDelayedExecutionCancelledJob() throws InterruptedException {
 
     StatusTestUtils.waitForStatus(node, Status.READY);
     // sleep so that job has time to get into delayExecution() -> wait()
-    Thread.sleep(1000L);
+    Thread.sleep(2000L);
```

Comment:
Is one second not long enough for the job to get into delayExecution()? I've never seen this test case failing in my local branch, so I'm wondering if this sleep time needs to be longer?

Reply:
Change at your own risk, but on Travis it's a whole different world. There are unexpected delays and in many places 1 second sleep hasn't been enough. Not saying this particular line would be as sensitive to random delays, but just trying to play it safe.